flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarListHandler.java

```diff
@@ -145,6 +155,7 @@ public boolean accept(File dir, String name) {
 					return writer.toString();
 				}
 				catch (Exception e) {
+					LOG.warn("Failed to fetch jar list.", e);
```

Comment:
Aren't failed completions logged anyways?

Reply:
not in legacy handlers

Reply:
scratch that, should be logged, I'll double check